### DISCUSSION TITLE
Web Version Redesign

### DISCUSSION THREAD

**Discussion by luni-moon** [pd4d10/git-touch#195] [Ideas]
I am putting the thread for the web redesign that I volunteered to do here, on #29. Please put all related suggestions and thoughts here, thanks!

**Comment by shreyas1599** [comment]
> Do you want me to change it to html and make a complete overhaul of the code, or just mike minor edits, I am assuming an overhaul on the design side, but just let me know, thanks! Also, could you open a discussion page on the web version, so it doesn’t clutter this “issue”, thanks!

Hi, flutter supports all platforms, so in line with that, this would require you to modify the existing repo and not create a separate one from scratch. In other words, the app should be [responsive](https://flutter.dev/docs/development/ui/layout/responsive). These may be helpful:

https://blog.codemagic.io/building-responsive-applications-with-flutter/
https://medium.com/flutter-community/the-best-flutter-responsive-ui-pattern-ba52875d70cd

It would be better if you start testing it on an iPad or any other tablet(simulator or device) because flutter on desktop/chrome is still primitive.

**Reply by luni-moon** [reply]
I realize that flutter supports all platforms, and I will need to not create a new repo from scratch, thanks for the reminder though, I already have a fork to test it on as well. I also have a couple of iPads as well, so I think I am all set, I assume by what you said, that you want a small remodel of the code, particularly style I still assume, correct?

**Reply by shreyas1599** [reply]
Yeah, that's correct.

**Reply by luni-moon** [reply]
I’ll get to work ASAP

**Reply by luni-moon** [reply]
My bad that I haven't gotten to it yet... I am gonna work on it right now. Sorry for the delay!

EDIT: @shreyas1599 I have a quick question. On CONTRIBUTING.md, it says something about using a Flutter Intl extension, is that for VSCode or what? Also, do I really need to do that since I am planning on changing style and HTML at this time? Thanks!

**Reply by luni-moon** [reply]
@shreyas1599, still want me to work on this? I have not seen any repo activity in a while.
